schema_utils.py

```diff
@@ -229,47 +259,118 @@ def get(cls, validator_id):
     def has_length_at_least(obj, min_value):
         """Returns True iff the given object (a list) has at least
         `min_value` elements.
+
+        Args:
+            obj: *. Object to evaluate.
+            min_value: int. Minimum value of elements.
+
+        Returns:
+            bool. True if the number of elements is greater than or equal
+                to min_value.
         """
         return len(obj) >= min_value
 
     @staticmethod
     def has_length_at_most(obj, max_value):
         """Returns True iff the given object (a list) has at most
         `max_value` elements.
+
+        Args:
+            obj: *. Object to evaluate.
+            max_value: int. Maximum value of elements.
+
+        Returns:
+            bool. True if the number of elements is less than or equal
+                to min_value.
         """
         return len(obj) <= max_value
 
     @staticmethod
     def is_nonempty(obj):
-        """Returns True iff the given object (a string) is nonempty."""
+        """Returns True iff the given object (a string) is nonempty.
+
+        Args:
+            obj: str. String to evaluate.
+
+        Returns:
+            bool. True if the object is not empty.
+        """
         return bool(obj)
 
     @staticmethod
     def is_uniquified(obj):
-        """Returns True iff the given object (a list) has no duplicates."""
+        """Returns True iff the given object (a list) has no duplicates.
+
+        Args:
+            obj: list. List to evaluate.
+
+        Returns:
+            bool. True if the object has no duplicates.
```

Comment:
bool. Whether the object has no duplicates.

(Ditto below. The rationale is that you've explained when True is returned, but not when False is. "Whether" captures both sides nicely.)

Reply:
Done